src/main/java/org/optaplanner/springbootcloudbalancing/solver/SolverTask.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.springbootcloudbalancing.solver;
+
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.api.solver.Solver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SolverTask<Solution_> implements Runnable {
+
+    private static final Logger logger = LoggerFactory.getLogger(SolverTask.class);
+
+    private final String tenantId;
+    private Solver<Solution_> solver;
+    private Solution_ planningProblem;
+
+    public SolverTask(String teantId, Solver<Solution_> solver, Solution_ planningProblem) {
+        this.tenantId = teantId;
+        this.solver = solver;
+        this.planningProblem = planningProblem;
+    }
+
+    @Override
+    public void run() {
+        logger.info("Running solverTask for tenantId {}.", tenantId);
```

Comment:
This will introduce duplicate logging: when the solver starts, there is already a "solver started" message. But it doesn't include the tenantId indeed... something we might want to think about. So maybe it's ok to leave it here for now.